### PR TITLE
PD-1121: Adds Toast guidelines to website

### PR DESCRIPTION
**Pull request by brookescarlett**
## ✍️ Proposed changes

🎟 _Jira ticket:_ [PD-1121](https://jira.mongodb.org/browse/PD-1121)

## 🛠 Types of changes

- [ ] Tooling (updates to workspace config or internal tooling)
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## ✅ Checklist

- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] I have run `yarn changeset` and documented my changes

- [ ] I have added my new package to the global tsconfig
- [ ] I have added my new package to the Table of Contents on the global README

### CODE REVIEW

File: website/pages/component/toast/guidelines.mdx [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+    display: flex;
+    justify-content: flex-start;
+    flex-grow: 1;
+    width: 100%;
+  `}
+>
+  <img
+    src={`${CDN}/images/guidelines/toast-3.png`}
+    alt=""
+    width="432px"
```
Comment (lazytype) [inline website/pages/component/toast/guidelines.mdx:142]: nit: move to css

File: website/pages/component/toast/guidelines.mdx [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+>
+  <img
+    src={`${CDN}/images/guidelines/toast-4.png`}
+    alt=""
+    width="432px"
+    className={css`
+      margin-left: -16px;
+    `}
+  />
+</div>
```
Comment (lazytype) [inline website/pages/component/toast/guidelines.mdx:167]: would other/future guidelines benefit from defining a component for this?

File: website/pages/component/toast/guidelines.mdx (hunk 1 above)
Comment (brookescarlett) [reply website/pages/component/toast/guidelines.mdx:142]: 👍

File: website/pages/component/toast/guidelines.mdx (hunk 2 above)
Comment (brookescarlett) [reply website/pages/component/toast/guidelines.mdx:167]: I'm not sure yet to be honest. Maybe we leave for now and as I build out more guidelines I will abstract out if necessary.

**Review by lazytype — APPROVED** [review]

**Pull request merged**